Library/Formula/miller.rb

```diff
@@ -0,0 +1,35 @@
+class Miller < Formula
+  desc "Miller is like sed, awk, cut, join, and sort for name-indexed data such as CSV."
```

Comment:
Please make this shorter, “`miller: <description>`” should fit under 80 characters.